indexing-service/src/main/java/org/apache/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -453,6 +454,64 @@ public void unregisterListener(String listenerId)
     return ImmutableList.copyOf(pendingTaskPayloads.values());
   }
 
+  @Override
+  public void enableWorker(String host)
+  {
+    sendRequestToWorker(host, "enable");
```

Comment:
looks like "enable" and "disable" strings are used at couple of places in this patch. Suggest to add an enum in `WorkerTaskRunner`  interface like
``` 
 enum ActionType{
    enable,
    disable
  }
```
and use that instead of strings to avoid errors.

Reply:
Done.